lib/slack.ml

```diff
@@ -186,21 +195,24 @@ let pp_list_with_previews ~pp_item list =
   end
   else List.map ~f:pp_item list
 
-let generate_push_notification notification channel =
+let generate_push_notification sender_slack_id notification channel =
```

Comment:
might be worth addming sender_slack_id either to the notification object or add a label `~sender_slack_id` since it's just a string.